package.json

```diff
@@ -114,6 +115,10 @@
     "start:beta": "NODE_ENV=development BETA=true webpack serve --config config/dev.webpack.config.js",
     "start:proxy": "PROXY=true npm run start",
     "start:proxy:beta": "BETA=true npm run start:proxy",
+    "translations": "npm-run-all translations:*",
+    "translations:extract": "npx formatjs extract ./src/*.js --out-file ./locales/translation-template.json",
+    "translations:compile": "npx formatjs compile ./locales/translation-template.json --out-file ./src/locales/translations.json",
```

Comment:
A note for a future, we will have to change this script to work for multiple files